### PR TITLE
Support this property

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Comment by cloud8little** [comment]
testing.

File: src/Neo.Compiler.CSharp/MethodConvert.cs [hunk 1]
```diff
@@ -2376,16 +2446,31 @@ private void ConvertElementAccessPreIncrementOrDecrementExpression(SemanticModel
...
+                AddInstruction(OpCode.DUP);
+                AddInstruction(OpCode.REVERSE4);
+                Call(model, property.SetMethod!, CallingConvention.Cdecl);
+            }
+            else
+            {
+                ConvertExpression(model, operand.Expression);
+                ConvertExpression(model, operand.ArgumentList.Arguments[0].Expression);
+                AddInstruction(OpCode.OVER);
+                AddInstruction(OpCode.OVER);
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert.cs:2466]: Move these lines outside the condition?

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:2466]: They are not shared parts, they just happen to be the same.

**Review by cloud8little — APPROVED** [review]

**Comment by cloud8little** [comment]
Test passed with #491

**Pull request merged**